### ISSUE TITLE
Remove plots

### ISSUE THREAD

**Issue by zyh722** [XMU-Kuangnan-Fang-Team/GENetLib#149]
I've noticed that when I run the code, it keeps printing the images of the test set and training set. I don't want these images to be displayed. What should I do?

**Comment by Barry57** [comment]
Thanks for your comment! You can view the coefficients in the function's plot by setting ```plot=False``` in ```scalar_ge```, and ```plot_res=False``` in ```func_ge```.
